packages/vscode-extension/package.json

```diff
@@ -132,7 +136,12 @@
         {
           "id": "teamsfx-empty-project",
           "name": "Teams Toolkit",
-          "when": "fx-extension.sidebarWelcome.default && !fx-extension.canUpgradeToArmAndMultiEnv"
+          "when": "fx-extension.sidebarWelcome.default && !fx-extension.canUpgradeToArmAndMultiEnv && !fx-extension.isInitAppEnabled"
+        },
+        {
+          "id": "teamsfx-empty-project-and-init",
+          "name": "Teams Toolkit",
+          "when": "fx-extension.sidebarWelcome.default && !fx-extension.canUpgradeToArmAndMultiEnv && fx-extension.isInitAppEnabled"
```

Comment:
@HuihuiWu-Microsoft please take a look, the new UX only works when the feature flags enabled and won't affect current UX.

Reply:
LGTM